Runtime/Tracking/Follow/Operation/Extraction/FollowerSourceExtractor.cs

```diff
@@ -0,0 +1,25 @@
+﻿namespace Zinnia.Tracking.Follow.Operation.Extraction
+{
+    using System;
+    using UnityEngine;
+    using UnityEngine.Events;
+    using Zinnia.Data.Operation.Extraction;
+
+    /// <summary>
+    /// Extracts the <see cref="GameObject"/> of the EventSource from a given <see cref="ObjectFollower.EventData"/>.
+    /// </summary>
+    public class FollowerSourceExtractor : GameObjectExtractor<ObjectFollower.EventData, FollowerSourceExtractor.UnityEvent>
```

Comment:
Name needs to match the class type it is for:

so should be `ObjectFollowerEventDataSourceExtractor` (same for the others too)

Reply:
Because i saw there is NotifierXXXExtractor which operates on a CollisionNotifier. So it is a hint for me to shorten the name. You still want the full name?

Reply:
Yeah they should be the full name, that's the correct way of referring to them (older extractors may not adhere to this, but that's because they're old)